x-pack/qa/repository-old-versions/src/test/java/org/elasticsearch/oldrepos/OldRepositoryAccessIT.java

```diff
@@ -130,10 +159,52 @@ public void testOldRepoAccess() throws IOException {
                 assertEquals(0, snapshotStatus.getShardsStats().getFailedShards());
                 assertThat(snapshotStatus.getStats().getTotalSize(), greaterThan(0L));
                 assertThat(snapshotStatus.getStats().getTotalFileCount(), greaterThan(0));
+
+                // so far only added basic infrastructure for reading 6.0.0+ indices
+                if (Build.CURRENT.isSnapshot() && oldVersion.onOrAfter(Version.fromString("6.0.0"))) {
+                    // restore index
+                    RestoreSnapshotResponse restoreSnapshotResponse = client.snapshot()
```

Comment:
Can we also test the restore and mount of old Lucene indices over closed indices too?

Reply:
I've extended the test to  cover this.